sky/provision/__init__.py

```diff
@@ -124,7 +124,8 @@ def delete_volume(provider_name: str,
 
 @_route_to_cloud_impl
 def get_volume_usedby(provider_name: str,
-                      config: models.VolumeConfig) -> List[str]:
+                      config: models.VolumeConfig,
+                      pod_only: bool = False) -> Tuple[List[str], List[str]]:
```

Comment:
While I can see `pod_only` is a optimization to reduce db calls, I feel it pollutes the provisioner API. Clouds may not have a notion of pods. Can we remove this flag, and always return clusters and external usedby (pods)?